sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -25,7 +25,7 @@ export interface AnomalyAlert {
     createdOn?: Date;
     id: string;
     modifiedOn?: Date;
-    timestamp?: Date;
+    timestamp?: number;
```

Comment:
Is this decision meant for all the packages?
(Asking for service-bus, I was hoping I'd change a "number" to "Date", but that doesn't seem like it?)

Reply:
@bterlson suggested that a timestamp should be a number or string